### DISCUSSION TITLE
Bank menu updating on deposit/withdrawal

### DISCUSSION THREAD

**Discussion by Evalutik** [BG-Software-LLC/SuperiorSkyblock2#1804] [Pending Feature Requests]
When I deposit or withdraw funds into the bank using the "deposit all" or "deposit half" button, the bank menu does not update automatically. You have to reopen the menu to see the changes. Please make this menu update after deposit or withdraw

**Comment by Lemvs** [comment]
@OmerBenGera it's fixed, you can close it
